modules/samples-operator-overview.adoc

```diff
@@ -64,7 +64,7 @@ For {product-title}, the default image registry is
 ifdef::openshift-enterprise[]
 `registry.redhat.io`.
 endif::[]
-ifdef::openshift-origin[]
+ifdef::openshift-rosa,openshift-dedicated[]
```

Comment:
```suggestion
ifdef::openshift-rosa,openshift-dedicated,openshift-origin[]
```
if we remove openshift-origin from this conditional, the OKD docs will have an incomplete sentence here. I think it should be re-added